mod-login-saml/src/main/resources/domain/mod-login-saml/features/loginSaml.feature

```diff
@@ -3,11 +3,55 @@ Feature: Login SAML
   Background:
     * url baseUrl
     * callonce login testUser
-    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
-
-  Scenario: SAML Check
-     Given path 'saml/check'
-     And header x-okapi-tenant = testTenant
-     And header x-okapi-token = '#(okapitoken)'
-     When method GET
-     Then status 200
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-tenant': #(testTenant), 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+  Scenario: Check endpoint missing okapi url header active is false
+    Given path 'saml/check'
+    When method GET
+    Then status 200
+    # A lot of the Rest Assured tests use the matchesJsonSchemaInClasspath method to validate JSON schema.
+    # Karate has a different opinion about how to validate JSON schema. See https://github.com/intuit/karate#schema-validation
+    # Here the only part of the schema returned is one property so all we can check is the data type of that property
+    # (this is karate's 'schema validation') and the value.
+    Then match response ==
+    """
+    {
+      active: '#boolean'
+    }
+    """
+    Then match response.active == false
+
+  # TODO In the rest assured tests this is 200 active true. But here it is active false 200. This is because
+  # the IdP isn't defined for the karate tests. We will need to configure IdP for the karate tests for this to work.
+  @Undefined
+  Scenario: Check endpoint has okapi url header active is true
```

Comment:
Similar to the first test case, this one would be `Check endpoint returns active is true when IdP is configured`

Reply:
Same. Agree this is language is better. Will make the change and merge.